### PR TITLE
Adding easing documentation

### PR DESCRIPTION
**Pull request by InventingWithMonster**

### CODE REVIEW

File: pages/easing.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  Callout,
+  Todo,
+} from "../components"
+
+export default Template
+
+# Easing
+
+<span className="lead">
+  A set of functions for adjusting the feel of tween
```
Comment (aron) [inline pages/easing.mdx]: Probably can just say “functions for adjusting the feel of animations” then explain what a tween is in the next section.

File: pages/easing.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  animations.
+</span>
+
+<div>
+
+[Tween animations](/api/animation/#tween) represent their progress as a value from `0` to `1`. If we pass this value to an easing function, we can adjust it to change the animation in different ways as it progresses.
```
Comment (aron) [inline pages/easing.mdx]: Suggestion: Tweening animations* represent progress from start to finish as a value between 0 and 1. A linear animation increments evenly from 0 to 1 through the duration of the animation. An easing function allows these numbers to be tweaked to create more natural or interesting transitions.

\* Is this term widely known enough, should we expand further or link to a long form article?

File: pages/easing.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+For instance, with the `Easing.expoOut` easing, we can start the animation fast and finish it slow. This kind of easing is useful for replicating natural motion.
```
Comment (aron) [inline pages/easing.mdx]: I think here (and in each of the examples) we could link to a site like https://easings.net/en to visualise the transition.

**Comment by eelco** [comment]
Closing because it somehow did not land, even though it’s good work. Please re-open when you start working on this.

**Pull request closed without merging**